docs/monitoring/containers.md

```diff
@@ -1,35 +1,266 @@
 title: Sematext Core Container Monitoring
-description: Monitor Docker, rkt and containerd containers with Sematext by installing simple Docker Certified Agents and using our interface that shows everything in one simple screen.
+description: Monitor Docker and containerd containers, container orchestration systems like kubernetes, swarm and nomad, or cloud container orchestration services EKS, ECS, AKS, GKE with Sematext by installing simple Docker Certified Agents and using our interface that shows everything in one simple screen.
 
-Monitor [Docker](https://www.docker.com/), rkt and containerd containers with Sematext by installing simple [Docker Certified Agents](https://hub.docker.com/_/sematext-agent-monitoring-and-logging) and using our interface that shows everything in one simple screen.
+Monitor [Docker](https://www.docker.com/) and containerd containers, container orchestration systems like kubernetes, swarm and nomad, or cloud container orchestration services EKS, ECS, AKS, GKE with Sematext by installing simple [Docker Certified Agents](https://hub.docker.com/_/sematext-agent-monitoring-and-logging) and using our interface that shows everything in one simple screen.
 
 ## Create a Sematext Monitoring App
-Creating a Sematext Monitoring App is as easy as choosing the Docker integration and giving the App a name.
+Creating a Sematext Monitoring App is as easy as choosing the Infra integration and giving the App a name.
 
 <video style="display:block; width:100%; height:auto;" controls autoplay>
-  <source src="https://cdn.sematext.com/videos/sematext-create-docker-app.mp4" type="video/mp4" />
+  <source src="https://cdn.sematext.com/videos/sematext-create-infra-app.mp4" type="video/mp4" />
```

Comment:
Do we need to wait for this video before publishing this doc?

Reply:
Yes, and this video waits for infra icon to be finished and added to SC. Did you have something else in mind?

Reply:
No, was just asking. Looks like the icon is done.